util/OptionsDB.cpp

```diff
@@ -137,8 +137,7 @@ OptionsDB::OptionsDB() {
     s_options_db = this;
 }
 
-void OptionsDB::Commit()
-{
+void OptionsDB::Commit() {
```

Comment:
I'll do as you say.

Can you understand what I am pointing out?
This line does not conform to what you and Geoff want for the style (bracket on the same line as the definition). Leaving this unchanged means the next rookie programmer who comes across this file might also come to the conclusion that the particular bracket stylistic choice is not important.